deno/payloads/v10/_interactions/_applicationCommands/permissions.ts

```diff
@@ -27,11 +27,11 @@ export interface APIGuildApplicationCommandPermissions {
  */
 export interface APIApplicationCommandPermission {
 	/**
-	 * The id of the role or user
+	 * The id of the role, user or channel
```

Comment:
```suggestion
	 * The id of the role, user or channel. Can also be a permission constant
```

Reply:
Is this documented on Discord's side? If not it should be

Reply:
It is yes
![image](https://user-images.githubusercontent.com/79790819/165779480-e82c4255-9aeb-42df-9e4a-7ea88e71973b.png)